src/textual/scrollbar.py

```diff
@@ -231,10 +231,14 @@ def render(self) -> RenderableType:
             style=scrollbar_style,
         )
 
-    async def on_enter(self, event: events.Enter) -> None:
+    def on_hide(self, event: events.Hide) -> None:
```

Comment:
If the scrollbar is hidden it releases the mouse. I'm wondering if this should be standard functionality, but there are some exotic use cases for a hidden widget capturing the mouse.